api.go

```diff
@@ -1365,7 +1365,7 @@ func updateOauthClient(keyName, apiID string, r *http.Request) (interface{}, int
 
 	// invalidate tokens if we had a new policy
 	if prevPolicy := client.GetPolicyID(); prevPolicy != "" && prevPolicy != updatedClient.PolicyID {
-		tokenList, err := apiSpec.OAuthManager.OsinServer.Storage.GetClientTokens(updatedClient.ClientID)
+		tokenList, _, err := apiSpec.OAuthManager.OsinServer.Storage.GetClientTokens(updatedClient.ClientID, 1)
```

Comment:
https://github.com/TykTechnologies/tyk/blob/c0dab5d1a023df456372b45795bd8936d6f369ba/api.go#L1366-L1369 was added in https://github.com/TykTechnologies/tyk/pull/2099 so might want to rethink by adding a new method that allows fetching all tokens still, or would passing a `-1` as the page parameter suffix - ideally no?

Reply:
why is it `GetClientTokens(updatedClient.ClientID, 1)` - that would return only 1st page of tokens to be wiped away. however, we still need to cleanup all tokens per client in case of policy change

Reply:
That is obviously wrong which is why I left the comment about a potential new method so all tokens can be wiped out. Just like the `GetClientTokensPaginated` you described in https://github.com/TykTechnologies/tyk/pull/2109#discussion_r257043315